<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>propagate_compute_dialog</class>
 <widget class="QWidget" name="propagate_compute_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>diaglog</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Resources/propagate.png</normaloff>Resources/propagate.png</iconset>
  </property>
  <widget class="QPushButton" name="propagate_compute">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>compute</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="propagate_front">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>propagate_front</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="propagate_back">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>propagate_back</string>
   </property>
  </widget>
  <widget class="QLabel" name="center_frame">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>centerframe</string>
   </property>
  </widget>
  <widget class="QLabel" name="front_endframe">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>front_endframe</string>
   </property>
  </widget>
  <widget class="QLabel" name="back_endframe">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>back_endframe</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="front_endframe_text">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>80</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="back_endframe_text">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>110</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="center_frame_text">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>20</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
